### ISSUE TITLE
Unable to refresh JWT auth token

### ISSUE THREAD

**Issue by GabeLambda** [box/box-python-sdk#197]
https://github.com/box/box-python-sdk/blob/master/boxsdk/auth/jwt_auth.py#L195

Callers of this method are expecting a tuple of (`access_token`, `refresh_token`), not just `access_token`. This breaks downstream in https://github.com/box/box-python-sdk/blob/master/boxsdk/session/box_session.py#L196, causing clients which use `JWTAuth` to break on refresh.

Example:
```
File "/usr/local/lib/python3.4/dist-packages/boxsdk/session/box_session.py", line 196, in _renew_session
    new_access_token, _ = self._oauth.refresh(access_token_used) 
ValueError: too many values to unpack (expected 2)
```

Version: 2.0.0a4

**Comment by maxdebayser** [comment]
This fix worked for me

**Comment by cschloer** [comment]
Fixed also worked for me but would rather not using a fork. Any status on this getting into master?

**Comment by jmoldow** [comment]
Fixed via #209

**Comment by jmoldow** [comment]
Also fixed in 1.5 via #217.

I just released a new version, 1.5.4, with this fix.
